tsconfig.json

```diff
@@ -7,6 +7,8 @@
     "paths": {
       "@/shared/*": ["./packages/shared/src/*"],
       "@/swap/*": ["./packages/swap/src/*"],
+      // This hack is because I was getting a type error in `vitest.shared` because ts used the wrong vite version.
```

Comment:
oh i think we actually have the same problem in the frontend 😅 so its not your fault